content/community/newsletter-archive.md

```diff
@@ -7,6 +7,7 @@ online: false
 
 2021
 
+- [October 2021](https://newsletters.getty.edu/t/t-1A51C0381B6ED3762540EF23F30FEDED): Quire at MuseumNext: Digital Collection Summit, New List of Digital Publishing Resources, Minneapolis Institute of Art
```

Comment:
"Collections" Summit

And include the bit about Contributing the the Docs and Website. That part is important.